delete-user-data/extension.yaml

```
@@ -148,6 +148,17 @@ params:
     default: shallow
     required: true
 
+  - param: SELECTED_DATABASE_INSTANCE
+    label: Selected Realtime Database instance
+    description: >
+      What is the selected Realtime Database used for this extension?
```

I'm not sure I understand the "selected" part. This param is used to select the instance, right?

How about:

```suggestion
  - param: SELECTED_DATABASE_INSTANCE
    label: Realtime Database instance
    description: >
      From which Realtime Database instance do you want to delete user data?
```